lib/mix/tasks/firmware.burn.ex

```diff
@@ -75,6 +75,8 @@ defmodule Mix.Tasks.Firmware.Burn do
            ask_pass = System.get_env("SUDO_ASKPASS") || "/usr/bin/ssh-askpass"
            System.put_env("SUDO_ASKPASS", ask_pass)
            {"sudo", ["fwup"] ++ args}
+        {_, :nt} ->
+           {"fwup", args}
```

Comment:
Is `fwup` working fine on Windows? I assume that you're using 0.9.0-dev. I'm planning on making an official `fwup` release really soon, but if there are any annoyances, now would be a good time for me to fix them.